gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -303,10 +302,10 @@ TFLiteCore::invoke (const GstTensorMemory * input, GstTensorMemory * output)
     TfLiteTensor *inputTensor = interpreter->tensor (in_tensor);
 
     switch (inputTensorMeta.info[i].type) {
-      case4type (_NNS_FLOAT32, float, f);
-      case4type (_NNS_UINT8, uint8_t, uint8);
-      case4type (_NNS_INT32, int, i32);
-      case4type (_NNS_INT64, int64_t, i64);
+        case4type (_NNS_FLOAT32, float, f);
```

Comment:
Looks good.

Reply:
1. This indentation is wierd
2. This is a style fix, which is not to be in a feature-add commit.